src/lib.rs

```diff
@@ -52,7 +52,8 @@ pub fn include_metadata(_: TokenStream) -> TokenStream {
                 footer: [u8; 4],
             }
 
-            #[link_section = ".metadata"]
+            #[cfg_attr(target_os = "macos", link_section = "__DATA,.metadata")]
```

Comment:
From our investigations, it seemed like the naming convention on Mac was `__DATA,__metadata`. Please can you double-check this is true and accept the suggestion if so?
```suggestion
            #[cfg_attr(target_os = "macos", link_section = "__DATA,__metadata")]
```

Reply:
@petekubiak ^